graphql/disablequery.go

```diff
@@ -0,0 +1,41 @@
+package graphql
```

Comment:
How do you feel about `QueryValidation` or something along those lines?

Reply:
I think our middleware could belong in a graphql-middleware package but I don't see a great benefit since the middleware is only used in one file so far, http_handler.go

Reply:
Sorry I should have clarified I was referring to file name lol

Reply:
I don't prefer QueryValidation because there is already an error code called "GRAPHQL_VALIDATION_FAILED" that is returned when the query doesn't validate against the gql schema. This check executes before our custom middleware.